egui/src/widgets/plot/items.rs

```diff
@@ -82,20 +82,26 @@ impl LineStyle {
                         }
                         shapes.push(Shape::line(line, stroke));
                     }
-                    LineStyle::Dotted { px } => {
+                    LineStyle::Dotted { spacing } => {
                         // Take the stroke width for the radius even though it's not "correct", otherwise
                         // the dots would become too small.
                         let mut radius = stroke.width;
                         if highlight {
                             radius *= 2f32.sqrt();
                         }
-                        points_from_line(&line, *px, radius, stroke.color, shapes)
+                        shapes.extend(Shape::dotted_line(&line, stroke.color, *spacing, radius))
                     }
-                    LineStyle::Dashed { px } => {
+                    LineStyle::Dashed { length } => {
                         if highlight {
                             stroke.width *= 2.0;
                         }
-                        dashes_from_line(&line, *px, stroke, shapes);
+                        let golden_ratio = (5.0_f32.sqrt() - 1.0) / 2.0; // 0.61803398875
```

Comment:
hint: `let golden_ratio = (5.0_f32.sqrt() + 1.0) / 2.0; // 1.61803398875` to avoid the division later :)

Reply:
True, but made redundant by the other change :)